.github/PR_Template.md

```diff
@@ -0,0 +1,20 @@
+Thank you for submitting this PR!
+
+**Please check if the PR fulfills these requirements**
+- [ ] The commit message follows our guidelines: `trans(title): topic1, topic2 or review(title): topic1, topic2`
```

Comment:
we should also have a commit message guideline markdown file so we can link it. #5